test/powershell/Modules/Microsoft.PowerShell.Utility/Select-String.Tests.ps1

```diff
@@ -78,6 +78,11 @@ Describe "Select-String" -Tags "CI" {
 	    $equal = @(Compare-Object $firstMatch $secondMatch).Length -eq 0
 	    $equal | Should -Be True
 	}
+
+	it "Should return a string type when -Raw is used" {
+		$result = $testinputtwo | Select-String -Pattern "hello" -CaseSensitive -Raw
+		,$result | Should -BeOfType "System.String"
```

Comment:
Why are you explicitly changing the left side to an array?

Reply:
Tbh: because the other tests that were already there were set up like that too, and I did not want to break style. I'll remove it. It does not matter for the outcome of the test though.